rest/blip_sync.go

```diff
@@ -959,7 +961,7 @@ func (ctx *blipSyncContext) setUseDeltas(clientCanUseDeltas bool) {
 
 	shouldUseDeltas := clientCanUseDeltas && ctx.sgCanUseDeltas
 	if !ctx.useDeltas && shouldUseDeltas {
-		ctx.dbc.DbStats.StatsDeltaSync().Add(base.StatKeyDeltaPullReplicationCount, 1)
+		ctx.db.DatabaseContext.DbStats.StatsDeltaSync().Add(base.StatKeyDeltaPullReplicationCount, 1)
```

Comment:
The 'DatabaseContext' should be optional here (and a few other spots above), since it's embedded in the db struct.